packages/create-plugin/src/utils/utils.config.ts

```diff
@@ -19,24 +21,49 @@ type UserConfig = {
 };
 
 export function getConfig(): CreatePluginConfig {
```

Comment:
`cosmiconfig` sounds like a good thing to explore, I didn't hear about it yet! But yeah, I was thinking the same, adding tests for these is crucial, as there are so many edge-cases that we will forget to test. (And as soon as we have tests it will be much easier to refactor as well, e.g. to use cosmiconfig.)

Reply:
Created an issue for it on our board: https://github.com/grafana/grafana-plugins-platform-team/issues/81